### DISCUSSION TITLE
which Mycroft version is being supported by OVOS

### DISCUSSION THREAD

**Discussion by Olzeke51** [OpenVoiceOS/ovos-buildroot#64] [General]
The initial READme indicates Mycroft 20.08 as being supported.
With the focus being Pi4 and the Mark2-pi devices -
could I assume that only Mycroft 21.2 and later will be the dev focus?
*
will the current BuildRoot makefile process - [as currrently documented] work on a pi3
*
Thank you for your time and answers - - times and software changes make it challenging to jump in.....
Gary

**Comment by JarbasAl** [comment]
the answer to this depends on what you mean by OVOS

our tools and skills should work in all mycroft-core versions, we test and develop in the current dev branch, so maybe some stuff may not work in master, but in that case open an issue in the relevant repos, eg, https://github.com/OpenVoiceOS/ovos_skill_manager

if you mean the OpenVoiceOS image (this repo), it is using https://github.com/HelloChatterbox/HolmesV which was synchronized with mycroft-core dev branch on 2021.6.30 

the version of HolmesV indicates the date of last sync. HolmesV includes some extra bugfixes and configuration options (eg, disabling msm, running offline) but has no new features, it is simply mycroft-core repackaged as a library (formerly called mycroft-lib)

either way the readme needs an update

**Reply by JarbasAl** [reply]
i should add we are targeting the pi4 during development, but once we have a stable image pi3 support will also be added, that is not a priority for now however
